### ISSUE TITLE
circleci: failed tests: TestTxnDBLostUpdateAnomaly_Delete

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#6388]
The following test appears to have failed:

[#17048](https://circleci.com/gh/cockroachdb/cockroach/17048):

```
I160428 17:01:53.053798 storage/replica_command.go:1439  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 1s following replica {0 0 0} 1970-01-01 00:00:00 +0000 UTC 0 [physicalTime=1970-01-01 00:00:00 +0000 UTC]
E160428 17:01:53.054753 config/config.go:382  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I160428 17:01:53.055376 storage/replica.go:1992  [node=1,store=1,range=1] gossiping sentinel from store 1, range 1
I160428 17:01:53.055669 storage/replica.go:1998  [node=1,store=1,range=1] gossiping first range from store 1, range 1
I160428 17:01:53.070401 kv/txn_correctness_test.go:562  attempting iso=[SERIALIZABLE SERIALIZABLE SERIALIZABLE] pri=[1 2 3] history=I1(A) C1 R2(A) W2(B-A) C2 D3(A) D3(B) C3
panic: can't increment key "A"; not yet read

goroutine 6617 [running]:
panic(0x17d0720, 0xc820d3d060)
	/usr/local/go/src/runtime/panic.go:464 +0x3ff
github.com/cockroachdb/cockroach/kv.incCmd(0xc820aa41c0, 0xc82000d1e0, 0xc820cac000, 0x0)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_correctness_test.go:199 +0x21b
github.com/cockroachdb/cockroach/kv.(*cmd).execute(0xc820aa41c0, 0xc82000d1e0, 0xc820cac000, 0x0, 0x0, 0x2aa722c)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_correctness_test.go:106 +0x1ca
github.com/cockroachdb/cockroach/kv.(*historyVerifier).runCmd(0xc820cd2000, 0xc82000d1e0, 0x1, 0x1, 0x0, 0xc820d3cfe0, 0x2, 0x2, 0xc820cac000, 0xc820ae7540)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_correctness_test.go:665 +0xa9
github.com/cockroachdb/cockroach/kv.(*historyVerifier).runTxn.func1(0xc82000d1e0, 0xc820ae7501)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_correctness_test.go:654 +0x71d
github.com/cockroachdb/cockroach/client.(*DB).Txn.func1(0xc82000d1e0, 0xc820ae7540, 0x4000000000000000)
--
	/go/src/github.com/cockroachdb/cockroach/kv/txn_correctness_test.go:659 +0x290
github.com/cockroachdb/cockroach/kv.(*historyVerifier).runHistory.func1(0xc820cd2000, 0xc820cd01e0, 0x3, 0x4, 0xc820cd02c0, 0x3, 0x3, 0xc820b6e2d0, 0xc820cac000, 0xc820111380, ...)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_correctness_test.go:577 +0x164
created by github.com/cockroachdb/cockroach/kv.(*historyVerifier).runHistory
	/go/src/github.com/cockroachdb/cockroach/kv/txn_correctness_test.go:580 +0xd86
FAIL	github.com/cockroachdb/cockroach/kv	38.407s
=== RUN   TestCombinable
--- PASS: TestCombinable (0.00s)
=== RUN   TestMustSetInner
--- PASS: TestMustSetInner (0.00s)
=== RUN   TestBatchSplit
--- PASS: TestBatchSplit (0.00s)
=== RUN   TestBatchRequestGetArg
--- PASS: TestBatchRequestGetArg (0.00s)
=== RUN   TestBatchRequestString
--- PASS: TestBatchRequestString (0.00s)

```
Please assign, take a look and update the issue accordingly.

**Comment by tschottdorf** [comment]
@spencerkimball just fixed this in #6386,  though it's not clear to me how the breaking change made it in.

**Comment by bdarnell** [comment]
It was write skew: #6353 changed the behavior of tests that use increment, and #6227 introduced a new test that used increment. #6353 was built off a 6-day-old commit, which is a long time for us. I still don't think we should turn on the github option that disallows all commit write skew, but i wish we could set some reasonable limit.
